client/src/client_sync/v18/network.rs

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: CC0-1.0
+
+//! Macros for implementing JSON-RPC methods on a client.
+//!
+//! Requires `Client` to be in scope.
+//!
+//! Specifically this is methods found under the `== Network ==` section of the
+//! API docs of Bitcoin Core `v0.18`.
+//!
+//! See, or use the `define_jsonrpc_minreq_client!` macro to define a `Client`.
+
+/// Implements Bitcoin Core JSON-RPC API method `getnodeaddresses`
+#[macro_export]
+macro_rules! impl_client_v18__getnodeaddresses {
+    () => {
+        impl Client {
+            pub fn get_node_addresses(&self, count: Option<u32>) -> Result<GetNodeAddresses> {
```

Comment:
While correct, just in general I have been excluding any optional args unless they are needed for integration testing. And later in the integration test you call this with `None`.

Reply:
I think I addressed this correctly, and do now remember you saying to leave out optional args, but it got lost in all the other info.